app/cloud_tasks/exceptions/__init__.py

```diff
@@ -0,0 +1,2 @@
+class CloudTaskCreationFailed(Exception):
```

Comment:
Can we standardise where we place custom exceptions?. For cloud tasks, it's within it's own exceptions module, for Pub/Sub it is within `exceptions.py` and for submitter it is a file for a specific exception. It would be nicer if we can make all these consistent. Personally, I think using `exceptions.py` in the working module is fine. This is what libs like redispy, google auth etc does.

Reply:
Moved these to an `exceptions.py`